docs/running.md

```diff
@@ -1,6 +1,10 @@
 # Running services
 
 Grid consists of many micro-services. There are a few different ways to start them.
+Note that some services don't compile until you hit them for the first time, once it's running you could try:
+
+for x in 900{1..9}; do curl http://localhost:$x; done
+for x in 901{0..11}; do curl http://localhost:$x; done
```

Comment:
hmmm, maybe you're right.  I've since discovered the Pingler which is a slightly nicer interface to this, was going to update this line to reflect.  Maybe the best advice would be, run the Pingler and use grid_runner to only start up the services you need, or to start them in a specific order?

btw, I have not had concurrency issues with sbt in grid, but maybe that's just because I wasn't doing too much work on it.